vllm_ascend/device_allocator/camem.py

```diff
@@ -60,7 +60,7 @@ def find_loaded_library(lib_name) -> Optional[str]:
     lib_name = find_loaded_library("vllm_ascend_C")
     camem_available = True
 except ImportError as e:
-    logger.error("Failed to import vllm_ascend_C:%s", e)
+    logger.warning("Failed to import vllm_ascend_C:%s", e)
```

Comment:
the info can be more clear that to let users know the sleep mode feature is disabled.